ui/src/EventForm.tsx

```diff
@@ -40,15 +36,21 @@ const EventForm = ({
   const [endTime, setEndTime] = useState<string>(initialEndTime);
   const [error, setError] = useState<string | null>(null);
   const [title, setTitle] = useState<string>(initialTitle);
-  const [description, setDescription] = useState<string>(initialDescription);
+  // `|| ""` prevents warning in tests
+  // "Warning: A component is changing an uncontrolled input of type text to be controlled."
+  const [description, setDescription] = useState<string>(
+    initialDescription || "",
+  );
```

Comment:
Sweet!

non-blocking (behavior is the same):

One other way to handle this is to set the initial value on the prop as it's getting passed in:

```
const EventForm = ({
  initialStartDate,
  initialEndDate,
  initialStartTime,
  initialEndTime,
  initialTitle,
  initialDescription = "", <<<<<<
  initialAllDay,
  initialRecurring,
  initialRecurrenceEnd,
  onFormSubmit,
  isCreate,
}: FormProps) => {
```

Reply:
Thanks, updated here and confirmed the test is still good 4acc3c3